alfasim_sdk/hook_specs.py

```diff
@@ -44,6 +44,53 @@ def compute_source_terms(
     """
 
 
+def calculate_slip_velocity(
+    ctx: "void*",
+    U_fields: "void*",
+    alpha_f: "void*",
+    d_disp_fields: "void*",
+    P: "void*",
+    rho_f: "void*",
+    mu_f: "void*",
+    sin_theta_f: "void*",
+    delta_x_f: "void*",
+) -> "int":
+    """
+    Internal simulator hook to calculate slip velocity between fluids
+    and solids.
```

Comment:
It is not clear which of those parameters are input and which are output...

Reply:
I will add extra explanation to docstring